vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -120,200 +94,236 @@ public static void main(String[] args) throws Exception {
             parser.printUsage(System.err);
             return;
         }
-        if (opts.arguments.isEmpty()) {
+
+        if (opts.xqFiles.isEmpty()) {
             parser.printUsage(System.err);
             return;
         }
+
         VXQuery vxq = new VXQuery(opts);
-        vxq.execute();
-        // if -timing argument passed, show the starting and ending times
-        if (opts.timing) {
```

Comment:
@prestoncarman: Based on the discussion me and @parshimers had few minutes ago, I think the execution time is an important metric. AsterixDB don't have this problem since they are using synchronous query requests while they also support asynchronous version which doesn't measure execution time accurately. In our case, since we are using 2 requests, one to submit query and one to fetch results, we have lost the link between execution times (since we are not waiting for hyracks to finish query execution).

On **repeated executions**: I have halted implementing that part because, as I understand, repeated executions are used for statistical purposes. Since we have a problem in measuring execution time, repeated executions doesn't mean anything at the moment. What do you think?

I will raise this question on the mailing list as well, so that we can discuss further on this.